flag/service/alertmanager/alertmanager.go

```diff
@@ -1,11 +1,10 @@
 package alertmanager
 
 type Alertmanager struct {
-	Address    string
-	BaseDomain string
```

Comment:
What is this `BaseDomain` and why is it being removed? I don't quite follow how this relates to the ingress changes?

Reply:
So I added it earlier because I was creating the ingress programmatically as they were in the installation-prometheus namespace but now as alertmanager is moved to the monitoring one, we can do without this so I removed it here as it was originally for this PR. Do you want me to split it out?

Reply:
Ah OK, I get it. No that's OK, with your explanation it makes sense.

Reply:
It's also my bad for not explaining :(